client/a8c-for-agencies/sections/marketplace/products-overview-v2/product-card/index.tsx

```diff
@@ -137,121 +130,98 @@ export default function ProductCard( props: Props ) {
 
-	const customDescription = useMemo( () => {
-		if ( product.slug === 'woocommerce-woopayments' ) {
-			return <WooPaymentsCustomDescription />;
-		}
-
-		return undefined;
-	}, [ product.slug ] );
-
-	const customFooter = useMemo( () => {
-		if ( product.slug === 'woocommerce-woopayments' ) {
-			return <WooPaymentsCustomFooter />;
-		}
-
-		return undefined;
-	}, [ product.slug ] );
-
-	const extraAsideContent = useMemo( () => {
-		if ( product.slug === 'woocommerce-woopayments' ) {
-			return <WooPaymentsRevenueShareNotice />;
-		}
-
-		return undefined;
-	}, [ product.slug ] );
-
-	const ctaLightboxLabel = useMemo( () => {
-		const selectedQuantity = quantity ?? 1;
-
-		if ( asReferral ) {
-			return isSelected ? translate( 'Remove from referral' ) : translate( 'Add to referral' );
-		}
-
-		if ( selectedQuantity > 1 ) {
-			return isSelected
-				? translate( 'Remove %(quantity)s from cart', { args: { quantity: selectedQuantity } } )
-				: translate( 'Add %(quantity)s to cart', { args: { quantity: selectedQuantity } } );
-		}
+	const isRedesign = isEnabled( 'a4a-product-page-redesign' );
 
-		return isSelected ? translate( 'Remove from cart' ) : translate( 'Add to cart' );
-	}, [ asReferral, isSelected, quantity, translate ] );
+	const hasMultipleProducts = products.length > 1;
 
-	const isRedesign = isEnabled( 'a4a-product-page-redesign' );
+	const productTitle = getProductShortTitle( currentProduct, hasMultipleProducts );
 
 	return (
-		<>
-			<div
-				onClick={ onSelect }
-				onKeyDown={ onKeyDown }
-				role="button"
-				tabIndex={ 0 }
-				aria-disabled={ isDisabled }
-				className={ clsx( {
-					'product-card': true,
-					selected: isSelected,
-					disabled: isDisabled,
-				} ) }
-			>
-				<div className="product-card__inner">
-					<div className="product-card__details">
-						<div className="product-card__main">
-							<div className="product-card__heading">
-								<h3 className="product-card__title">{ productTitle }</h3>
-								<ProductBadges product={ product } />
-								<div className="product-card__pricing is-compact">
-									<ProductPriceWithDiscount
-										product={ product }
-										hideDiscount={ hideDiscount }
-										quantity={ quantity }
-										compact
-									/>
-								</div>
-
-								<div className="product-card__description">
-									{ truncateDescription( productDescription ) }
-								</div>
+		<div
+			className={ clsx( 'product-card', customProductCard?.className, {
+				'product-card--with-variant': hasMultipleProducts,
+				selected: isSelected,
+				disabled: isDisabled,
+			} ) }
+			onKeyDown={ onKeyDown }
+			onClick={ onSelect }
+			role="button"
+			aria-disabled={ isDisabled }
+			tabIndex={ 0 }
+		>
+			<div className="product-card__inner">
+				<div className="product-card__details">
+					<div className="product-card__main">
+						<div className="product-card__heading">
+							{ customProductCard?.image && <img src={ customProductCard?.image } alt="" /> }
```

Comment:
[Non-blocker]

```suggestion
							{ customProductCard?.image && <img src={ customProductCard?.image } alt="Product Logo" /> }
```

or we could add an `alt in `useCustomProductCard`